colors/monokai.vim

```diff
@@ -87,7 +87,7 @@ hi erubyDelimiter ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui
 hi erubyComment ctermfg=95 ctermbg=NONE cterm=NONE guifg=#75715e guibg=NONE gui=NONE
 hi erubyRailsMethod ctermfg=81 ctermbg=NONE cterm=NONE guifg=#66d9ef guibg=NONE gui=NONE
 hi htmlTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
-hi htmlEndTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
+hi htmlEndTag ctermfg=NONE ctermbg=NONE cterm=NONE guifg=#a6e22e guibg=NONE gui=NONE
```

Comment:
Yeah I know it looks awkward, however, it seems like the HTML start tag "inherited" (if that's the right word in this context) its color from `Function`, however, the HTML end tag didn't.

#### Original config:
![screen shot 2015-08-03 at 11 07 44 am](https://cloud.githubusercontent.com/assets/2080075/9044214/057acb5a-39d0-11e5-84c5-c14a1fbf4a29.png)

#### Proposed change:
![screen shot 2015-08-03 at 11 06 25 am](https://cloud.githubusercontent.com/assets/2080075/9044199/f5b8fe58-39cf-11e5-848e-de8b71e3a6e6.png)